.github/workflows/tact.yml

```diff
@@ -119,6 +119,36 @@ jobs:
           then false
           fi
 
+      - name: CLI Test | Check compilation of broken contract doesn't contain stacktrace
+        if: runner.os != 'Windows'
+        run: |
+          output=$(tact src/test/compilation-failed/contracts/const-eval-ascii-empty.tact 2>&1 || true)
+          if echo "$output" | grep -E ".*at \w+ \(.*"; then
```

Comment:
Should we include `at Object.run (` here to make the pattern more robust?

Reply:
I don't think we can call this type of testing robust in any case, since we need a proper way to test the CLI :)

Reply:
Also for constant evaluation errors there is no `Object.run`

Reply:
Ah, nice catch!